src/FutureJs.re

```diff
@@ -26,6 +26,7 @@ let fromPromise = (promise, errorTransformer) =>
          |> ignore
          |> Js.Promise.resolve
        )
+    |> ignore
```

Comment:
Does this modify the interface?

Reply:
yes it does. Ok I'll change this PR to ensure backwards compatibility